apps/dashboard/src/hooks/use-create-workflow.ts

```diff
@@ -34,6 +40,13 @@ export function useCreateWorkflow({ onSuccess }: UseCreateWorkflowOptions = {})
 
       onSuccess?.();
     },
+
+    onMutate: () => {
+      showSavingToast(setToastId);
+    },
```

Comment:
why do we need to show two toasts here always? one for saving and another for success?
let's just have success or error toast

Reply:
good point! I missed the duplication toast.